nixos/modules/services/networking/wireguard.nix

```diff
@@ -160,6 +160,14 @@ let
         interval of 25 seconds; however, most users will not need this.'';
       };
 
+      table = mkOption {
+        default = "main";
+        type = types.str;
+        description = ''The kernel routing table to add this peer's associated
```

Comment:
Something I didn't realize first time through. Why is this per-peer? Shouldn't this be per interface?

I realize there's an argument to be made for granularity, but it's hard to imagine what use cases for that you're imagining. Wouldn't it be simpler to do this per-interface instead?

Or do you have a particular per-peer use case in mind?